core/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -204,7 +205,7 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
 
-    @Override
+    @Override @SwallowsExceptions(reason = "?")
     public String toString() {
```

Comment:
One mindless way to solve the problem with methods that look like this is to forward them all to `Strings.toString` and just annotate that one with `@SwallowsException(reason = "We don't want a broken toString to bubble up")`. I'm not sure that is a **good** reason, but it is the reason the `toString`s all do this.